source/_common.R

```diff
@@ -17,11 +17,13 @@ knitr::opts_chunk$set(
   fig.width = 6,
   fig.asp = 0.618,  # 1 / phi
   fig.show = "hold",
-  eval = NA,
-  echo = NA
+  eval = NULL,  # Emphasizing these are the defaults.
```

Comment:
The reason I switched away from using NULL is because that does not trigger the hook (hooks only run when options are set).